CHANGELOG.md

```diff
@@ -93,6 +93,7 @@ Shipping methods can be removed by the user after it has been assigned to a chec
   - Add `CheckoutLine.problems` field
   - Add `CheckoutSettingsInput` to `ChannelCreateInput` & `ChannelUpdateInput`
   - Add `checkoutSettings` field to `Channel`
+- Add Filter warehouses by metadata -#13345 by @Smit-Parmar
```

Comment:
Missing space

Reply:
@8r2y5 Above lines are sub points. You can verify that by opening full file.

Reply:
Done!